app/setup.php

```diff
@@ -96,6 +96,21 @@
 add_action('wp_ajax_update_favorites', 'App\\update_favorites');
 add_action('wp_ajax_nopriv_update_favorites', 'App\\update_favorites');
 
+/**
+ * Set language cookies when settings form is submitted.
+ */
+add_action('init', function () {
+    if (isset($_POST['track_viewed_resources'])) {
```

Comment:
When the form is submitted, a "Settings successfully saved" confirmation notification should appear.

Reply:
sounds good too!